common.tf

```
@@ -42,6 +42,7 @@ locals {
   enable_apiary_s3_log_management = var.apiary_log_bucket == "" ? true : false
   enable_apiary_s3_log_hive       = var.apiary_log_bucket == "" && var.enable_apiary_s3_log_hive ? true : false
   apiary_s3_logs_bucket           = local.enable_apiary_s3_log_management ? "${local.apiary_bucket_prefix}-s3-logs" : ""
+  apiary_s3_logs_checkpoint_bucket= "${local.apiary_bucket_prefix}-s3-logs-checkpoint"
```

Add a space before the `=` and then align the rest of the assignments too please 
```suggestion
  apiary_s3_logs_checkpoint_bucket = "${local.apiary_bucket_prefix}-s3-logs-checkpoint"
```